### PR TITLE
Add field Company.one_list_tier

### PR DESCRIPTION
**Pull request by marcofucci**
### Description of change

This:
- adds a new field `Company.one_list_tier` with fk to OneListTier
- populates the `one_list_tier` field from the `classification` field
- makes the field `classification` read-only in the Django Admin
- populates the `classification` field from `the one_list_tier` field when a company is saved via the Django Admin

The last point happens on the Django Admin save instead of the model to avoid accidentally overriding the values.

### Checklist

* [x] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Have any relevant search models been updated?
* [x] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [x] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/company/migrations/0048_populate_one_list_tier_field.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('company', '0047_one_list_tier_field'),
+    ]
+
+    operations = [
+        migrations.RunPython(populate_one_list_tier_field, migrations.RunPython.noop),
```
Comment (reupen) [inline datahub/company/migrations/0048_populate_one_list_tier_field.py]: Shouldn't this be marked as elidable? As in, we won't need it after squashing migrations?

Comment (marcofucci) [reply datahub/company/migrations/0048_populate_one_list_tier_field.py]: We need it as the initial fixture is in the migration file and this data migration only moves data from that table to this one. If I add `elidable`, the resulting table will be empty, no?

Comment (reupen) [reply datahub/company/migrations/0048_populate_one_list_tier_field.py]: I might be missing something – it copies data from `classification_id` to `one_list_tier_id` on the companies table, no?  After squashing, what would it be copying?

Comment (marcofucci) [reply datahub/company/migrations/0048_populate_one_list_tier_field.py]: oh yeah it's this migrations :/ sorry you are right, was thinking about the other migration

**Pull request merged**